packages/upset/src/components/ImportModal.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useContext, useState } from 'react';
+import { Alert, Button, Dialog, DialogContent, DialogTitle, Snackbar } from '@mui/material';
+import { ProvenanceContext } from './Root';
+
+export const ImportModal = (props:{open: boolean, close: () => void}) => {
+  const [ isError, setIsError ] = useState(false);
+  const { actions } = useContext(
+    ProvenanceContext,
+  );
+
+  async function readFile(file: File) {
+    const data = await file.text();
+  
+    const newState = JSON.parse(data);
```

Comment:
We will eventually need this to be HJSON compatible. We should make an issue or just fix it now. 1 library and a changed call here